tests/README.md

```diff
@@ -4,8 +4,7 @@ Examples are tested using tool [htrun](https://github.com/ARMmbed/mbed-os-tools/
 
 To run the test, use following command after you build the example:
 ```
-mbedhtrun -d D: -p COM4 -m K64F -f .\BUILD\K64F\GCC_ARM\mbed-os-example-for-google-iot-cloud.bin --compare-log tests\google-iot-cloud.log
+mbedhtrun -d D: -p COM4 -m K64F -f .\BUILD\K64F\GCC_ARM\mbed-os-example-for-google-iot-cloud.bin --compare-log tests\google-iot-cloud.log --baud-rate=115200 --sync=0
```

Comment:
@rajkan01 I think you have used `-P 120` option as well

Reply:
Yes it is required some time but better to specify this -P 120 option here

Reply:
It is only required some time but better to specify this -P 120 option here

Reply:
I'll add it in case the server is slow to reply

Reply:
Added